### PR TITLE
Fix typescript strip in class properties pass

### PR DESCRIPTION
**Pull request by Brooooooklyn**

### CODE REVIEW

File: ecmascript/transforms/src/compat/es2020/class_properties.rs [hunk 1]
```diff
@@ -586,7 +586,10 @@ impl ClassProperties {
                         c.params = c.params.move_map(|mut param| match &mut param {
                             ParamOrTsParamProp::TsParamProp(p) => match &p.param {
                                 TsParamPropParam::Ident(i) => {
-                                    typescript_constructor_properties.push(store(i));
+                                    typescript_constructor_properties.push(store(&Ident {
+                                        type_ann: None,
```
Comment (kdy1) [inline ecmascript/transforms/src/compat/es2020/class_properties.rs:590]: I think `type_ann` will be dropped by typescript_strip pass, isn't it?

File: ecmascript/transforms/src/compat/es2020/class_properties.rs [hunk 2]
```diff
@@ -595,11 +598,19 @@ impl ClassProperties {
                                 }
                                 TsParamPropParam::Assign(pat) => match &*pat.left {
                                     Pat::Ident(i) => {
-                                        typescript_constructor_properties.push(store(i));
+                                        typescript_constructor_properties.push(store(&Ident {
+                                            type_ann: None,
```
Comment (kdy1) [inline ecmascript/transforms/src/compat/es2020/class_properties.rs:602]: Same as above

**Comment by kdy1** [comment]
Anyway, the change looks harmless to me. Thanks

File: ecmascript/transforms/src/compat/es2020/class_properties.rs (hunk 1 above)
Comment (Brooooooklyn) [reply ecmascript/transforms/src/compat/es2020/class_properties.rs:590]: No, seems like `type_ann` has not been dropped

Comment (kdy1) [reply ecmascript/transforms/src/compat/es2020/class_properties.rs:590]: Ok, I got it.

**Review by kdy1 — APPROVED** [review]

File: ecmascript/transforms/src/compat/es2020/class_properties.rs (hunk 1 above)
Comment (Brooooooklyn) [reply ecmascript/transforms/src/compat/es2020/class_properties.rs:590]: I got this output:

```js
class BadRequestError extends Error {
    constructor(message){
        super(message);
        this.message: string = message;
    }
}
```

from

```ts
class BadRequestError extends Error {
  constructor(public readonly message: string) {
    super(message)
  }
}
```

**Pull request merged**